test/containers/check-bastion

```diff
@@ -18,17 +18,17 @@
 # along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
 
 
-from testvm import VirtMachine
-from testlib import *
-from verify import parent
 import os
 import sys
 
 test_dir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
 sys.path.append(test_dir)
 
+from verify import parent
```

Comment:
... looking at "Revert "python files: fix header include order"", that now explains how tests could actually work. :grin: Please eliminate this commit pair.